File: testprojects/master_scala_mainone/pipelines/SCALA_COMPLEX/code/src/main/scala/graph/Reformat_4.scala
package graph

import io.prophecy.libs._
import config.ConfigStore._
import udfs.UDFs._
import udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_4 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.select(
      lookup("Lookup_2", col("p_string"), col("p_long"))
        .getField("c_date-for today")
        .as("col1")
    )

}

File: testprojects/master_python_mainone/pipelines/DONOT_OPEN_PYTHON/code/job/graph/Reformat_4.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def Reformat_4(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(lit(10).alias("c_short"), lit(123231).alias("c_int"))
